CHANGELOG.md

```diff
@@ -334,3 +334,13 @@ Released with 1.0.0-beta.37 code base.
 1. The function `outputBigNumberFormatter` in `web3-core-helper` renamed to `outputBigIntFormatter` under `web3-common`
 2. Removed `this.defaultBlock` context from `inputDefaultBlockNumberFormatter` in `web3-core-helper` and converted to additional parameter
 3. Removed `this.defaultBlock` context from `inputTransactionFormatter` in `web3-core-helper` and converted to additional parameter
+
+#### web3-utils
+
+1. The following functions `soliditySha3` `soliditySha3Raw` `encodePacked` now includes type validation and requires type specficiation, instead of guessing the value type
+2. The functions `soliditySha3` `soliditySha3Raw` `encodePacked` does not support BN and now supports `BigInt`
+3. The functions `flattenTypes` and `jsonInterfaceMethodToString` moved to the `web3-eth-abi` package
+
+### web3-eth-accounts
+
+1. `create` function does not take in the optional parameter `entropy`
```

Comment:
If we are not including optional `ignoreLength` in `privateKeyToAccount` like 1.x [doc](https://web3js.readthedocs.io/en/v1.5.2/web3-eth-accounts.html#privatekeytoaccount) it should be mentioned here. Its not in 1.x code but in doc